### ISSUE TITLE
Handle negative numbers, and floating points

### ISSUE THREAD

**Issue by dustmop** [dustmop/xc#1]

**Comment by dustmop** [comment]
Negative numbers implemented in 482b628.

**Comment by dustmop** [comment]
Floating points should work like this:

```
> xc 5 / 2
0x02.80   2.5   0x02 rem 0x01   2 rem 1
```

```
> xc 7.25 + 6.5
0x0d.c0   13.75
```

**Comment by dustmop** [comment]
Floating points implemented by f46ec3c58a86d429395da40ce9ea3c6ed1eff173.
